docs/platform/stackscripts-new-manager/index.md

```diff
@@ -0,0 +1,307 @@
+---
+author:
+  name: Alex Fornuto
+  email: afornuto@linode.com
+description: 'Create Custom Instances and Automate Deployment with StackScripts.'
+keywords: ["ami", "automation", "elasticity", "cloud", "custom instance"]
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+aliases: ['stackscripts/']
+modified: 2018-08-22
+modified_by:
+  name: Linode
+published: 2011-04-05
+title: Automate Deployment with StackScripts
+external_resources:
+  - '[StackScript Community Library](http://linode.com/stackscripts)'
+hiddenguide: true
+---
+
+![Automate Deployment with StackScripts](automate-deployment-with-stackscripts-title-graphic.jpg "Automate Deployment with StackScripts")
+
+[StackScripts](http://linode.com/stackscripts/) provide Linode users with the ability to automate the deployment of custom systems on top of our default Linux distribution images. Linodes deployed with a StackScript run the script as part of the first boot process. This guide explains how StackScripts work, and offer several examples of how to use them.
+
+## Introduction
+
+StackScripts are usually Bash scripts, stored in the Linode Manager, and can be accessed when you deploy a Linode. During the first boot job of the newly created disks, the StackScript will run, using any variable you may have added, and perform the scripted commands.
+
+## Deploying from a StackScript
+
+1.  Start by clicking the **Create** button and selecting **Linode**.
+
+    ![Create a new Linode](stackscripts-create-a-linode.png "Create a new Linode")
+
+2.  Navigate to the **Create from StackScript** tab.
+
+    ![Navigate to Create from StackScript.](stackscripts-navigate-to-create-from-stackscript.png "Navigate to Create from Stackscript.")
+
+3.  At this point you have the option of using a a Linode StackScript, Community StackScript, or one of your own.
+
+    [![StackScript selection options.](stackscripts-selection-screen-small.png "StackScript selection options.")](stackscripts-selection-screen.png)
+
+<!-- Alternatively, you can follow along with this video, which will show you how to deploy from a StackScript:
+
+{{< youtube XviHHyXVH20 >}} -->
+
+### Using a Community StackScript
+
+{{< note >}}
+There is no default logging when using a StackScript. Output is sent to the console.
+{{< /note >}}
+
+1.  In addition to the StackScripts created by Linode, any community member who creates a StackScript has the option to make it publicly available to other customers. As seen in the image below, you can use the search field to narrow your search to match your distribution and goal:
+
+    [![A search for 'wordpress.'](stackscripts-search-wordpress-small.png "A search for 'wordpress.'")](stackscripts-search-wordpress.png)
```

Comment:
Not a big deal: this section is called (and is about) community SS's, but the image shows the Linode tab selected.